lib/include/hse_ikvdb/tuple.h

```diff
@@ -55,14 +56,14 @@ struct kvs_ktuple {
  * instead encodes a special value (e.g., a tomb).
  */
 struct kvs_vtuple {
-    void *vt_data;
-    u64   vt_xlen;
+    void *   vt_data;
+    uint64_t vt_xlen;
```

Comment:
Ugh, please align the asterisks with the field name..

Reply:
if only we had clang-format to save us :'(. Will do.